### DISCUSSION TITLE
Managed to get connect to discord RPC but fails to set RPC.

### DISCUSSION THREAD

**Discussion by Central1234** [Radiicall/jellyfin-rpc#109] [Q&A]
Hi, just wondering if its possible to get some help. i've managed to get the script to pick up my emby server and connect to discord. but whenever i go play something i get this error:

Failed To Set Activity
Error: The Pipe is being closed. (os error 232)

then it just re attempts to connect to RPC which it does successfully in 1 try but then attempts to set the RPC again and just goes in a loop

Would love to some help!
Thanks!

**Comment by justradical** [comment]
Moved to #110
